public/client.js

```diff
@@ -0,0 +1,26 @@
+const socket = io();
+
+const connectionCount = document.getElementById('connection-count');
+const statusMessage = document.getElementById('status-message');
+const buttons = document.querySelectorAll('#survey-choices button');
+const countedVotes = document.getElementById('results');
+
+socket.on('voteCount', function (votes) {
```

Comment:
I'm seeing duplication in regards to the part I mentioned above. So, it's definitely a good time to refactor this stuff out.